x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -95,17 +109,75 @@ interface Props {
   updatePageState?: (update: Partial<ListingPageUrlState>) => void;
 }
 
+export const getModelStateColor = (
```

Comment:
This file is pretty big, could this function be moved to a utils file?

Reply:
Refactored in https://github.com/elastic/kibana/pull/167407/commits/beef167d516e35179858be75a40ca33d0b221775